lib/bin/run_test.js

```diff
@@ -63,6 +63,23 @@ var _ = require('lodash');
 var co = require('co');
 var exit = require('exit');  // process.exit that works on Windows
 var promiseUtil = require('../promise_util');
+var cleaned = false;
+
+function cleanup(exitCode) {
+  if (cleaned) {
+    return;
+  }
+  cleaned = true;
+  require('ps-tree')(process.pid, function (err, children) {
+    var childProcesses = children.map(function (p) { return p.PID; });
+    childProcesses.forEach(function(pid) {
+      if (require('is-running')(pid)) {
+        process.kill(pid);
```

Comment:
This will send a SIGTERM to the child process. It is not certain that the child process will actually quit when it receives just a SIGTERM. You may want to use `soft_kill.js` in the same folder.